phi/playground/playground.py

```diff
@@ -136,6 +136,20 @@ def agent_chat(body: AgentRunRequest):
                 run_response = cast(RunResponse, agent.run(body.message, stream=False))
                 return run_response.model_dump_json()
 
+        @playground_routes.post("/agent/sessions/all")
+        def get_agent_sessions():
+            session_data = []
+            for agent in self.agents:
+                session_data.append(agent.get_session_data())
+            return session_data
+
+        @playground_routes.post("/agent/sessions/{session_id}")
```

Comment:
@manthanguptaa 
1. Let's add a `body: ReadAgentSessionRequest` that includes an `agent_id` and `user_id`
2. Let's make a call to `.read_from_storage()` to read the data from storage first
3. Let's return the AgentSession object

So the code will look like:
```
agent: Optional[Agent] = None
for _agent in self.agents:
    if _agent.agent_id == body.agent_id:
        agent = _agent(session_id=session_id)
        break

agent.read_from_storage()
return agent.to_agent_session()
```

Reply:
agent = _agent(session_id=session_id) 
why are we making a call like this?

Reply:
```
agent: Optional[Agent] = None
for _agent in self.agents:
    if _agent.agent_id == body.agent_id:
        agent = _agent
        break

agent.session_id = session_id
agent.read_from_storage()
return agent.to_agent_session()
```